test/JDBC/expected/non_default_server_collation/japanese_ci_as/test_like_for_AI-vu-verify.out

```diff
@@ -2815,6 +2815,7 @@ SELECT * FROM test_like_for_AI_prepare_t1_cs WHERE col_c LIKE 'cafe';
 GO
 ~~START~~
 nvarchar#!#varchar#!#text#!#ntext#!#char#!#nchar
+café#!#café#!#café#!#café#!#café                                              #!#café                                              
```

Comment:
This should not be the case as per our expectations. As char adds extra padding based on the typmod, in PG, they are not same. We so not return anything is case of CI_AS and CS_AS. So, are we okay with this?

Reply:
As discussed, this is intended T-SQL behaviour so we will go with this.